libcsl_godot/Cargo.toml

```diff
@@ -18,3 +18,4 @@ rand = "0.8.5"
 pkcs5 = {version = "0.7.1", features = ["pbes2", "alloc"]}
 scrypt = "0.11.0"
 cbor_event = "2.1.3"
+cardano-message-signing = "=1.0.1"
```

Comment:
We were talking about vendoring it into the project, as it was not published to officially (but this dependency also works).